### PR TITLE
update swa version

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: cli/azd/pkg/tools/swa/swa_test.go [hunk 1]
```diff
@@ -60,10 +60,10 @@ func Test_SwaBuild(t *testing.T) {
 
 			require.Equal(t, "./projectPath", args.Cwd)
 			require.Equal(t, []string{
-				"-y", "@azure/static-web-apps-cli@1.0.6",
+				"-y", "@azure/static-web-apps-cli@1.1.1",
 				"build",
 				"--app-location", "service/path",
-				"--output-location", "build",
+				"--output-location", "build", "--verbose=silly",
```
Comment (wbreza) [inline cli/azd/pkg/tools/swa/swa_test.go:66]: I'm assuming we want to keep this level of verbosity in the logs?

**Review by wbreza — COMMENTED** [review]
Does upgrading to 1.1 fix any of the issue we saw when running in Github actions?

**Comment by vhvb1989** [comment]
> Does upgrading to 1.1 fix any of the issue we saw when running in Github actions?

I have created this PR to test that.. I wanted to have our CI to create and push a docker img that I can use for the CI build where I am testing the issues.
the swa cli repo requires folks to try latest version before creating issues :).  So,, there I go

**Comment by vhvb1989** [comment]
No need for this anylonger.
swa-cli issue created: https://github.com/Azure/static-web-apps-cli/issues/688

**Pull request closed without merging**